### PR TITLE
ApplicationInsights logging in NLog and log4net should include ExceptionTelemetry.Message

### PR DESCRIPTION
**Pull request by snakefoot**
Non-breaking change for log4net and NLog.

Will improve this screenshot to include Message-details for Exceptions:

image

### CODE REVIEW

File: LOGGING/test/Log4NetAppender.Net45.Tests/ApplicationInsightsAppenderTests.cs [hunk 1]
```diff
@@ -325,6 +325,7 @@ public void CustomMessageIsAddedToExceptionTelemetryCustomProperties()
             }
 
             ExceptionTelemetry telemetry = (ExceptionTelemetry)this.appendableLogger.SentItems.First();
+            Assert.IsTrue(telemetry.Message.Contains("Test logging exception"));
```
Comment (cijothomas) [inline LOGGING/test/Log4NetAppender.Net45.Tests/ApplicationInsightsAppenderTests.cs]: the Message contains not just the "Test logging exception" but the exception type as well right? Can you validate taht as well in tests

Comment (snakefoot) [reply LOGGING/test/Log4NetAppender.Net45.Tests/ApplicationInsightsAppenderTests.cs]: Sure

**Comment by snakefoot** [comment]
> Could you update changelog as well with this information?

Done

**Comment by TimothyMothra** [comment]
@snakefoot Can you please update `Changelog.md` under the VNext section? 

I'm sorry for the inconvenience and I've removed the older Changelogs to eliminate this confusion. These were left over from the repo migration.

This is our final ask and I'm ready to merge this now.

**Comment by snakefoot** [comment]
> Can you please update Changelog.md under the VNext section?

Done

**Comment by TimothyMothra** [comment]
@snakefoot We published 2.12-Beta4 this morning including your changes! :)

**Pull request merged**